### ISSUE TITLE
howto: types without globals

### ISSUE THREAD

**Issue by jvanattedev** [UI5/linter#937]
Following code is reported as an error:
```
statusTimestamp.bindProperty('text', {
  path: 'Timestamp',
  type: 'sap.ui.model.odata.type.DateTimeOffset',
  formatOptions: {
    pattern: 'dd.MM.yyyy HH:mm:ss',
    UTC: true
  }
})
```

It is reported as an error: Access of global variable 'sap.ui.model.odata.type.DateTimeOffeset' etc.
How can this type be written in another way?
